flake.nix

```
@@ -0,0 +1,23 @@
+{
+  inputs = {
+    holonix.url = "github:holochain/holochain";
```

```suggestion
    holonix.url = "github:holochain/holochain";
    holonix.inputs.versions.url = "github:holochain/holochain?dir=versions/0_1";
```

not necessary but makes it more predictable. otherwise you could end up switching to 0_2 without opting into it explicitly